tests/conftest.py

```diff
@@ -1,101 +1,47 @@
-import os
-
 import pytest
-import imp
 
 
-def pytest_addoption(parser):
-    default = 'scenario.py'
-    parser.addoption(
-        "--scenario",
-        action="store",
-        default=default,
-        help="YAML file defining scenarios to test. Currently defaults to: %s" % default
+@pytest.fixture()
+def CephNode(Ansible, Interface, Command, request):
```

Comment:
It's capitalized to match the rest of the ``testinfra`` fixtures, but doesn't really need to be. I doubt we'll ever operate on a node that's not ceph.